transports/janus_mqtt.c

```diff
@@ -377,6 +401,43 @@ int janus_mqtt_init(janus_transport_callbacks *callback, const char *config_path
 		ctx->admin.publish.topic = NULL;
 	}
 
+	/* Status messages configuration */
+	janus_config_item *status_enabled_item = janus_config_get(config, config_status, janus_config_type_item, "enabled");
+	if(status_enabled_item && status_enabled_item->value && janus_is_true(status_enabled_item->value)) {
+		ctx->status.enabled = TRUE;
+
+		janus_config_item *status_connect_message_item = janus_config_get(config, config_status, janus_config_type_item, "connect_message");
+		if(status_connect_message_item && status_connect_message_item->value) {
+			ctx->status.connect_message = g_strdup(status_connect_message_item->value);
+		} else {
+			ctx->status.connect_message = NULL;
```

Comment:
I don't think you need the `else`s here, `ctx` is created via `g_malloc0` which also initializes everything to `0` and `NULL` by default.